engine/src/main/java/org/terasology/rendering/dag/nodes/FirstPersonViewNode.java

```diff
@@ -69,13 +103,9 @@ public void process() {
             playerCamera.updateMatrices();
             playerCamera.loadProjectionMatrix();
 
-            /**
-             * Resets the state after the render of the First Person View.
-             */
-            GL11.glDepthFunc(GL_LEQUAL);
+            //GL11.glDepthFunc(GL_LEQUAL);
             GL11.glPopMatrix();
```

Comment:
The GL11.* and the playerCamera.* statements are what I was talking about in the Forum @Josharias: perhaps we could remove them. But right now I'd be more tempted to remove the whole node frankly.